### PR TITLE
Revert "include tests in installed package (#232)"

### PR DESCRIPTION
**Pull request by perrinjerome**
This reverts commit 84a5015efcb4ad510febefa66654ebc60473c10a and commit 48165265cba59e7ef9110f2a1646e0d9e706aea3.

RestrictedPython intentionally does not include tests anymore

See discussion on https://github.com/zopefoundation/RestrictedPython/issues/231 and https://github.com/zopefoundation/RestrictedPython/pull/232

### CODE REVIEW

File: tests/transformer/test_yield.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def test_yield_from():
     def my_external_generator():
         my_list = [1, 2, 3, 4, 5]
         for elem in my_list:
-            yield elem
+            yield(elem)
```
Comment (loechel) [inline tests/transformer/test_yield.py:37]: i am not sure if that was done by you or any code formater, but it should stay:

```python
yield elem
```

File: tests/transformer/test_functiondef.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ def test_RestrictingNodeTransformer__visit_FunctionDef__7():
     assert result.errors == (functiondef_err_msg,)
 
 
+
```
Comment (loechel) [inline tests/transformer/test_functiondef.py:38]: i guess that will become a linting warning, only two empty lines between blocks

File: tests/transformer/test_yield.py (hunk 1 above)
Comment (perrinjerome) [reply tests/transformer/test_yield.py:37]: When doing #232 we moved the tests in `src/` folder, so the linters from https://github.com/zopefoundation/RestrictedPython/blob/48165265cba59e7ef9110f2a1646e0d9e706aea3/tox.ini#L59-L60 applied, so we changed the code to pass the lint steps at the same time.

This commit was done automatically with  `git revert` command so it brings us back to the "before" state.

If we want to apply linters to the `tests` folder, that's another story, the linter commands are generated from https://github.com/zopefoundation/meta/blob/b5df3766ff8923477f3d24729b19504f0c401a2e/config/default/tox-lint.j2#L7-L8 and they don't cover `tests` folder. If we want to include lint in `tests` folder it seem better to me to treat this in a separate issue and limit the scope here to just revert.

**Review by icemac — APPROVED** [review]
Let's just plainly revert #232 for now.
I am currently working on linting the tests.

**Comment by icemac** [comment]
> I am currently working on linting the tests.

See #234 as my proposed solution.

**Pull request merged**